cmd/kube-apiserver/app/server.go

```diff
@@ -444,7 +444,9 @@ func BuildGenericConfig(s *options.ServerRunOptions, proxyTransport *http.Transp
 		return nil, nil, nil, nil, nil, fmt.Errorf("invalid authentication config: %v", err)
 	}
 
-	genericConfig.Authorizer, genericConfig.RuleResolver, err = BuildAuthorizer(s, sharedInformers)
+	genericConfig.Authorizer, genericConfig.RuleResolver, err =
```

Comment:
nit: don't make function calls and definitions span multiple lines like this? other reviewers haven't commented on this, but it's not really how the rest of the code base looks.